system-config/skip-encrypted-swap.rules

```diff
@@ -0,0 +1,17 @@
+SUBSYSTEM!="block", GOTO="qubes-rule-end"
+ACTION=="remove", GOTO="qubes-rule-end"
+
+# Skip devices that might contain untrusted data.  We won’t be doing any other
+# processing on them (and rightly so).
+KERNEL=="loop*", ATTR{loop/backing_file}!="/var/lib/qubes/vm-kernels/*", ENV{SYSTEMD_READY}="1"
+ENV{DM_LV_NAME}=="vm-*", ENV{SYSTEMD_READY}="1"
+ENV{DM_UUID}=="CRYPT-*", ENV{DM_NAME}!="swap|luks-*", ENV{SYSTEMD_READY}="1"
+ENV{DM_NAME}=="vm-*", ENV{SYSTEMD_READY}="1"
+
+# Process swap device.  We only skipped them so that Qubes would not show
+# them as available for export.
+ENV{DM_NAME}=="swap", ENV{DM_UUID}=="CRYPT-PLAIN-swap", ENV{DM_UDEV_DISABLE_DISK_RULES_FLAG}=""
```

Comment:
This assumes a specific device name, but such device is not created anywhere in the qubes code base, nor this assumed name is documented anywhere. Are you going to submit a separate PR that configures encrypted swap?

Reply:
Indeed I will.  I should probably split the parts needed for encrypted swap into separate PRs.